google-api-client/src/main/java/com/google/api/client/googleapis/services/AbstractGoogleClientRequest.java

```diff
@@ -239,6 +247,25 @@ public final boolean getDisableGZipContent() {
     return this;
   }
 
+  /**
+   * Sets whether the response should return raw input stream or not.
+   *
+   * <p>
+   * By default ti is {@code false}.
+   * </p>
+   *
+   * <p>
+   * Normally response stream is wrapped GZipInputStream when it contains content-encoding header
```

Comment:
Maybe:

When the response contains a known content-encoding header, the response stream is wrapped with an InputStream that decodes the content. This fails when we download large files in chunks (see #bugnumber). Setting this to true will make the response return the raw input stream.